AmazonChimeSDKDemo/AmazonChimeSDKDemo/AppConfiguration.swift

```diff
@@ -11,4 +11,6 @@ import Foundation
 struct AppConfiguration {
     static let url = "YOUR_SERVER_URL"
     static let region = "YOUR_SERVER_REGION"
+    static let broadcastBundleId = "YOUR_BROADCAST_BUNDLE_ID"
```

Comment:
is this optional or not putting it will not run the app?

Reply:
optional, only needed if builder wants to test broadcast